src/AsideBody/AsideBody.js

```diff
@@ -3,29 +3,34 @@ import React, { Component } from 'react';
 import './AsideBody.css';
 import CardList from '../CardList/CardList.js';
 import FaveList from '../FaveList/FaveList.js';
+import WishList from '../WishList/WishList.js';
 
 function AsideBody(props) {
   if (props.asideView === "myCardList") {
     return (
       <div>
-        <CardList cardNames={props.cardNames} 
-                  removeListItem={props.removeListItem}
-                  setCardCount={props.setCardCount}
-                  userCardsData={props.userCardsData} 
-                  compareBuilds={props.compareBuilds}/>
+        <CardList 
+          cardNames={props.cardNames} 
+          removeListItem={props.removeListItem}
+          setCardCount={props.setCardCount}
+          userCardsData={props.userCardsData} 
+          compareBuilds={props.compareBuilds}/>
       </div>
     )
   } else if (props.asideView === "faveDecks") {
     return (
       <div>
-        <FaveList userFaveDecks={props.userFaveDecks}
-                  removeFaveListItem={props.removeFaveListItem}/>
+        <FaveList 
+          userFaveDecks={props.userFaveDecks}
+          removeFaveListItem={props.removeFaveListItem}/>
       </div>
     )
   } else if (props.asideView === "wishList"){
     return (
       <div>
-        <p>Wish List Goes HERE!</p>
+        <WishList 
```

Comment:
This might just be trivial stylistic preferences, but what do you think about a single return with short circuited && operators instead of if, else if, else if? It would be similar to how we did it in Card.js.